docs/libraries.md

```diff
@@ -1,52 +1,144 @@
+compiled language, cross-compilation is _not_ required to run code on ev3dev,
+unlike other compiled languages; this makes Go much simpler to work with.
+
+[Get started with Go](https://github.com/ev3go/ev3dev){: .btn .btn-default }
+
+## C++
+<https://github.com/ddemidov/ev3dev-lang-cpp>
+
+C++ is a low-level, compiled language which is highly performant while still
+providing modern language features. It is best for applications which require
+the fastest execution or interaction with existing C++ libraries.
+
+[Get started with C++](https://github.com/ddemidov/ev3dev-lang-cpp){: .btn .btn-default }
+
+## C
+<https://github.com/in4lio/ev3dev-c>
+
+C is a low-level, compiled language which is useful for interacting with other
+C-based code. It is very lightweight and often the most portable across platforms.
+
+[Get started with C](https://github.com/in4lio/ev3dev-c){: .btn .btn-default }
+
+
+## Vala, Genie and other GObject-based languages with ev3devKit
+<https://github.com/ev3dev/ev3devKit>
+
+Through [GObject-introspection](https://wiki.gnome.org/Projects/GObjectIntrospection),
+this library can be used by languages including Vala and Genie, among
+[many others](https://wiki.gnome.org/Projects/GObjectIntrospection/Users). This
+is great for people who want to use higher-level syntax while still producing a
+performant application, or people who want a less error-prone API for C. The
+[Brick Manager][https://github.com/ev3dev/brickman] for ev3dev is written using
+this library.
+
+[Get started with ev3devKit](https://github.com/ev3dev/ev3devKit){: .btn .btn-default }
+
+<br>
+<br>
+
+# Alternative implementations and wrappers
+
+Some of the libraries above also implement interfaces for other languages. If
+you're looking for an alternative implementation for any reason, try out the
+options below.
+
+## Python, Ruby and Perl with ev3dev-c
+<https://github.com/in4lio/ev3dev-c>
+
+ev3dev-c also has wrappers for [Python](https://github.com/in4lio/ev3dev-c/tree/master/python),
+[Ruby](https://github.com/in4lio/ev3dev-c/tree/master/ruby) and [Perl](https://github.com/in4lio/ev3dev-c/tree/master/perl). 
+
+## Python with ev3devKit
+<https://github.com/ev3dev/ev3devKit>
+
+Ev3devKit also has a Python wrapper. You can find demos of using ev3devKit from
+Python [here](https://github.com/ev3dev/ev3devKit/tree/ev3dev-jessie/demo/python).
+
+# Out-of-date, abandoned and unfinished implementations
+
+<div class="panel panel-danger">
+<div class="panel-heading">
+    {% include icon.html type="danger" %}
+    Listings below this point are for libraries that are not up-to-date, have
+    been abandoned, or are unfinished.
+</div>
+<div class="panel-body">
+<p>
+    Use them with caution, as some functionality will likely be broken. If you
+    see a library below that you'd like to see in a better state, consider
+    contributing to it to get it updated and ready to use.
+</p>
+</div>
+</div>
+
+## C#
```

Comment:
The `#` in `C#` is not showing up on the preview site. You might need to escape it (`C\#`)